.github/workflows/lint_test_charts.yaml

```diff
@@ -66,3 +89,56 @@ jobs:
         run: ct install --namespace ct --config .github/ct-install.yaml
       - name: Test upgrade charts
         run: ct install --namespace ct --config .github/ct-install.yaml --upgrade
+
+  bundle-integrity:
+    name: Test nri-bundle dependencies has the same version of the common-library
+    runs-on: ubuntu-latest
+    if: ${{ needs.lint-test.outputs.bundle-changed == 'true' }}
+    needs: [ lint-test ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: azure/setup-helm@v3
+        with:
+          version: 'v3.0.0'
+
+      - name: Add helm repositories
+        run: |
+          helm repo add newrelic-helm-charts https://newrelic.github.io/helm-charts
+          helm repo add newrelic-cdn-helm-charts https://helm-charts.newrelic.com
+          helm repo add newrelic-infrastructure https://newrelic.github.io/nri-kubernetes
+          helm repo add nri-prometheus https://newrelic.github.io/nri-prometheus
+          helm repo add nri-metadata-injection https://newrelic.github.io/k8s-metadata-injection
+          helm repo add newrelic-k8s-metrics-adapter https://newrelic.github.io/newrelic-k8s-metrics-adapter
+          helm repo add kube-state-metrics https://kubernetes.github.io/kube-state-metrics
+          helm repo add nri-kube-events https://newrelic.github.io/nri-kube-events
+          helm repo add pixie-operator-chart https://pixie-operator-charts.storage.googleapis.com
+          helm repo add newrelic-infra-operator https://newrelic.github.io/newrelic-infra-operator
+
+      - name: Build nri-bundle dependencies
+        run: helm dependency update ./charts/nri-bundle
+
+      - name: Decompress nri-bundle and all its dependencies recursively
+        run: |
+          while [ `find charts/nri-bundle -name \*.tgz -or -name \*.tar.gz | wc -l` -gt 0 ]; do
+            find charts/nri-bundle -name \*.tgz -or -name \*.tar.gz | while read line; do
+              echo "Decompressing $line..."
+              tar xzf "$line" -C $(dirname "$line")
+              echo "Removing $line..."
+              rm "$line"
+            done
+          done
```

Comment:
When you use `helm package` (that the chart releaser uses) it packages the dependency uncompressed but it could not be the case ins case somebody does something manually and we start supporting external charts or something weird. That is the reason I prefer to do not trust charts we are depending even if they are from us.